libs/generic-view/ui/src/lib/interactive/modal/helpers/modal-size-controller.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { APIFC } from "generic-view/utils"
+import { withConfig } from "../../../utils/with-config"
+import { ModalSize } from "./modal-content"
+
+interface Config {
+  size: ModalSize
+}
+
+export const ModalSizeControllerSmall = styled.div`
+  display: none;
+`
+
+export const ModalSizeControllerMedium = styled.div`
+  display: none;
+`
+
+export const ModalSizeControllerLarge = styled.div`
+  display: none;
+`
+
+export const ModalSizeController: APIFC<undefined, Config> = ({ config }) => {
```

Comment:
⚪ (question)
s this component a placeholder for when designs for other sizes are introduced? From my understanding, it currently doesn’t perform any function, correct?

Reply:
I see: https://github.com/mudita/mudita-center/pull/1832/files#diff-eb75d6ebce3f39c3f0a2bb8ddd2e35b9956330d5c4aefc05f241f19635174657R53

Reply:
Why is the modal size specified by a dedicated component instead of props? If it's more convenient for you, let's catch up on a sync. Maybe you could help me understand this idea.